lib/chef/azure/service.rb

```diff
@@ -6,9 +6,48 @@ class ChefService
   include ChefAzure::Shared
   AZURE_CHEF_SERVICE_PID_FILE = "azure-chef-client.pid"
   AZURE_CHEF_CRON_NAME = 'azure_chef_extension'
+  DEFAULT_CHEF_SERVICE_INTERVAL = 30
+  CLIENT_RB_INTERVAL_ATTRIBUTE_NAME = 'interval'
+

```

Comment:
Not sure a separate card is necessary but thats up to your discression. If the method is not called by `EnableChef` I'd make it private. Thanks.